bluechi.spec.in

```diff
@@ -81,6 +85,10 @@ Summary:  BlueChi service controller agent
 Requires: systemd
 Recommends: bluechi-selinux
 
+%if 0%{?with_coverage}
+Requires: bluechi-coverage = %{version}-%{release}
```

Comment:
Why would everyone in the world want bluechi-coverage installed?  At most this should be a suggests.

Reply:
If you build the RPMs with coverage support (by default we build without it), then we could assume that you want to install RPMs providing coverage data, right?

Reply:
And I've added Requires with specific version/release to make sure that installed coverage data matches the installed binaries

Reply:
> If you build the RPMs with coverage support (by default we build without it), then we could assume that you want to install RPMs providing coverage data, right?

I agree. And the only use case that makes sense setting `with_coverage=1` are the integration tests (at the moment). This is probably the best approach considering our test setup to get the coverage results, but I am also not happy to have a package purely for testing.